packages/ui-predicate-react/src/default-components/logical-types.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+class LogicalTypes extends Component {
```

Comment:
You are not managing state inside this component, only props. You could use functionnal component

Reply:
You are right. I will change it.